src/net.js

```diff
@@ -278,7 +278,7 @@ class Network {
        * The URL of the operation.
        * @param {string} rel
        * The type of operation. For example: `preconnect`.
-       * @param {boolean} [crossorigin]
+       * @param {boolean} [cor] crossorigin
```

Comment:
The original form of this was correct (i.e. crossorigin). By changing this to "cor", the name of the attribute  in the documentation will be changed!